lib/engines/hcktest/hcktest.json

```diff
@@ -35,6 +35,39 @@
             "value": "161"
           }
         ]
+      },
+      {
+        "tests": [
+          "[NdisStudio] NdisPoll keyword functional validation",
+          "[NdisStudio] OidRequestTest - OID_GEN_VENDOR_DRIVER_VERSION Verification",
+          "[NdisStudio] NdisPoll keyword validation",
+          "[NdisStudio] OidRequestTest - OID_GEN_XMIT_OK and OID_GEN_RCV_OK Verification",
+          "[NdisStudio] OidRequestTest - OID_GEN_VENDOR_ID Verification"
+        ],
+        "pre_test_commands": [
+          {
+              "desc": "Disable NTP service",
+              "guest_run": "Stop-Service w32time"
```

Comment:
Not just stop you also need to disable service. HLK can reboot the PC during tests, so the service will start again.